packages/esm-patient-clinical-view-app/src/enhanced-adherence/enhanced-adherence.components.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { usePatientEnhancedAdherence } from '../hooks/usePatientEnhancedAdherence';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  DataTable,
+  TableContainer,
+  TableHeader,
+  TableExpandHeader,
+  TableExpandRow,
+  TableExpandedRow,
+  StructuredListSkeleton,
+} from '@carbon/react';
+import { ErrorState } from '@openmrs/esm-patient-common-lib';
+import AdherenceTable from './adherence-table.component';
+type EnhancedAdherenceProps = {
+  patientUuid: string;
+};
+
+const EnhancedAdherence: React.FC<EnhancedAdherenceProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+  const headerTitle = t('enhancedadherence', 'Advanced Enhanced Adherence');
```

Comment:
```suggestion
  const headerTitle = t('enhancedadherence', 'Enhanced Adherence');
```
to have consistency same as☝🏽